www/_posts/2016-03-02-ios-4.1.0.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Cordova iOS 4.1.0"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova iOS 4.1.0` has been released!
```

Comment:
Perhaps we should highlight CB-10530 and one of the fixes is that this improves stability significantly. There were lots of reports about this one.